assets/src/edit-story/components/button/index.js

```diff
@@ -37,6 +39,24 @@ const Base = styled.button.attrs(
 	` }
 `;
 
+const StyledButton = styled( Base )`
+	border: none;
+	padding: 0;
+	width: ${ ( { width } ) => width }px;
+	height: ${ ( { height } ) => height }px;
+	min-width: initial;
+	visibility: ${ ( { isHidden } ) => isHidden ? 'hidden' : 'visible' };
+	opacity: .25;
```

Comment:
Nitpick: I think the figma uses .3 opacity in general for inactive elements. But maybe it varies.

Reply:
Eh, completely likely. Will change for consistency.